corgidrp/sorting.py

```diff
@@ -0,0 +1,232 @@
+import copy
+import numpy as np
+
+import corgidrp.data as data
+
+def extract_frame_id(filename):
+    """
+    Extract frame ID from an L1 filename. Structure is assumed to be ending
+    like '..._frame_id.fits' where frame_id is a series of digits
```

Comment:
is it confirmed that the last element of a filename will be a frame ID?

Reply:
The plan is to be compatible with IIT. Now, it is. This is a comment for the next version

Reply:
I'll resolve this once I have the time to post an updated list of to-do's in https://github.com/roman-corgi/corgidrp/issues/226